magefile.go

```diff
@@ -348,12 +348,15 @@ func BuildOpaBundle() error {
 		URL:             fmt.Sprintf("https://github.com/%s/%s.git", owner, r),
 		InsecureSkipTLS: true,
 	})
+	if err != nil {
```

Comment:
Once [CI Fix PR](https://github.com/elastic/cloudbeat/pull/662) is merged this code can be reverted.

Reply:
Checking for errors is already a good practice imo